doc/docs/master/how-tos/export-data-out-pachyderm/mount-repo-to-local-computer.md

```diff
@@ -0,0 +1,187 @@
+first download the files to your computer, edit them,
+and then put them back to the repository. The `pachctl mount`
+command automates all these steps for you. You can mount just the
+`images` repo or all Pachyderm repositories as directories
+on you machine, edit as needed, and, when done,
+exit the `pachctl mount` command. Upon exiting the `pachctl mount`
+command, Pachyderm uploads all the changes to the corresponding
+repository.
+
+## Avoiding Merge Conflicts
+
+If someone else modifies the files while you are working on them
+locally, their changes will likely be overwritten when you exit
+`pachctl mount`. This happens because your changes are saved to
+the Pachyderm repository only after you interrupt the `pachctl mount`
+command. Therefore, make sure that you do not work on the
+same files while someone else is working on them.
+
+Also, if you have any automated way
+to upload files to a Pachyderm repo, and you are modifying files
+on your local computer you might overwrite these changes with
+your changes, when you terminate the `pachctl mount` command.
+
+## Access Permissions
+
+By default, the `pachctl mount` command mounts a directory with
+`READ` access only. You can use the `--write` flag with the command
+to enable `WRITE` access. However, even if you mount all the repos
+with `WRITE` access, you cannot modify the files in the
+Pachyderm output repositories. Because these repositories are
+created by the Pachyderm pipelines, they are immutable. Only a pipeline
+can change and update files in these repositories. If you try to change
+a file in an output repo, you will get an error message.
+
+## Mounting Branches and Commits
+
+The `pachctl mount` command allows you to mount not only the default
+branch, typically a `master` branch, but also other Pachyderm
+branches. By default, Pachyderm mounts the `master` branch. However,
+if you add a branch to the name of the repo, the `HEAD` of that branch
+will be mounted. 
+
+**Example:**
+
+```bash
+pachctl mount images --repos images@staging+w
+```
+
+You need to add the `+w` to the branch to enable write access.
+
+Also, you can mount a specific commit. Because commits
+might be on multiple branches, modifying them might result in data deletion
+in the `HEAD` of the branches. That is why you can only mount commits in
+read-only mode.
+
+## Prerequisites
+
+You must have the following configured for this functionality to work:
+
+* Unix or Unix-like operating system, such as Ubuntu 16.04 or macOS
+Yosemite or later.
+* FUSE and SSHFS for your operating system installed:
+
+  * On macOS, run:
+
+    ```bash
+    brew cask install osxfuse
+    ```
+
+    ```bash
+    brew install sshfs
+    ```
+
+  * On Ubuntu, run:
+
+    ```bash
+    sudo apt-get install sshfs
+    ```
+
+    For more information, see:
+
+    * [FUSE for macOS](https://osxfuse.github.io/)
+    * [FUSE on Ubuntu](https://gist.github.com/cstroe/e83681e3510b43e3f618)
```

Comment:
Why the link to this gist?

Reply:
Definitely not the linked one, because it includes a bunch of errata that aren't relevant, like sshfs and zfs. Maybe the manpage? https://manpages.ubuntu.com/manpages/precise/man8/mount.fuse.8.html

Reply:
This page does not have information about how to actually install fuse on Ubuntu. I don't have Ubuntu to test, but I think `sudo apt-get install -y fuse` should do the trick. I'll remove the link.